.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Report a bug in Dapr's Java Pluggable Components SDK
+title: ''
+labels: kind/bug
```

Comment:
Make sure to define the labels from the issues page.

Reply:
Interesting that you mentioned that and we already have 'Bug' and other labels by default. Curious why we went for the `kind` prefix

Reply:
I asked @artursouza  and he said that this `kind/` prefix was something they picked up from other OSS projects and that this setup is being used by dapr/dapr. I renamed the labels accordingly so this merge should work just fine.